src/components/Reports/FourteenMonthReports/Layout/Table/Table.test.tsx

```diff
@@ -261,6 +261,29 @@ describe('FourteenMonthReportTable', () => {
     expect(queryByTestId('FourteenMonthReport')).toBeInTheDocument();
   });
 
+  it('should should show the dot if a donation is late', async () => {
+    const { getAllByTestId, queryByTestId } = render(<Components />);
+
+    await waitFor(() => {
+      expect(
+        queryByTestId('LoadingFourteenMonthReport'),
+      ).not.toBeInTheDocument();
+    });
+
+    const fourteenMonthReportRow = getAllByTestId(
+      'FourteenMonthReportTableRow',
+    );
+    expect(
+      within(fourteenMonthReportRow[0]).getByTestId('lateCircle60'),
+    ).toBeInTheDocument();
+
+    await waitFor(() => {
+      expect(
+        within(fourteenMonthReportRow[1]).queryByTestId('lateCircle30'),
+      ).not.toBeInTheDocument();
+    });
```

Comment:
```suggestion
    expect(
        within(fourteenMonthReportRow[1]).queryByTestId('lateCircle30'),
    ).not.toBeInTheDocument();
```

Reply:
I thought we usually have `waitFor` when querying something that should not be in the document. No?

Reply:
Yes, that's a good rule of thumb for something that needs to asynchronously load but we expect to not be in the document. However, in this case, we already waited for the table rows to load, so at that point, either the late circle will be there or it won't. There's no reason to wait longer.

Reply:
Ah got it